tools/osbuilder/rootfs-builder/nvidia/nvidia_rootfs.sh

```diff
@@ -280,6 +281,8 @@ chisseled_init() {
 	ln -sf  /bin/NVRC init
 	ln -sf  /bin/NVRC sbin/init
 
+	cp -a "${stage_one}"/"${libdir}"/libgcc_s.so.1*    "${libdir}"/.
```

Comment:
This is not needed anymore with the latest NVRC which does not rely on libgcc.

Reply:
@zvonkok Yeah, I also thought that we could move to `${machine_arch}-unknown-linux-musl`, but unfortunetaly the `s390x-unknown-linux-musl` target is still tier 3 in rust.

It seems to me that having this patch would be easier than building the `s390x-unknown-linux-musl` target only for s390x

Reply:
I found s390x in other places in kata, so I thought we were planning to support it. Sorry about that.

No problem. I have patches to statically build the NVRC, I can push a PR for that. I also had to bump upx because it fails to pack a statically built NVRC (`CantPackException: circular DT_HASH chain 1`).

Thanks